plugins/out_stackdriver/stackdriver_conf.c

```diff
@@ -281,6 +298,38 @@ struct flb_stackdriver *flb_stackdriver_conf_create(struct flb_output_instance *
         ctx->resource = flb_sds_create(FLB_SDS_RESOURCE_TYPE);
     }
 
+    flb_kv_init(&ctx->labels);
+
+    /*
+     * Load monitored resource labels from config files.
+     * Will only use labels specified for a particular resource type at
+     * https://cloud.google.com/logging/docs/api/v2/resource-list
```

Comment:
I'm a bit confused by this comment. How does it know what labels to use without hard-coding them or reading the canonical list from somewhere? Did you mean to say that labels that don't match the particular resource type at that URL will be rejected by the API?

Reply:
I meant to say the latter. Will update with that.

Reply:
They do, that is my oversight. Will fix.